scripts/pre_commit_linter.py

```diff
@@ -384,6 +386,15 @@ def _get_all_files_in_directory(dir_path, excluded_glob_patterns):
     return files_in_directory
 
 
+@contextlib.contextmanager
+def _redirect_stdout(new_target):
+    old_target, sys.stdout = sys.stdout, new_target
```

Comment:
Separate this into 2 lines so it's easier to read.

Reply:
Done, thanks!